app/src/main/java/org/mozilla/fenix/ext/TabPartition.kt

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ext
+
+import mozilla.components.browser.state.state.TabPartition
+
+/**
+ * Check if a [TabPartition] has no tabs
+ *
+ * @return true if the [TabPartition] has no tabs, false otherwise.
+ */
+fun TabPartition?.isEmpty(): Boolean {
```

Comment:
Let's move these to A-C as well and add to https://github.com/mozilla-mobile/android-components/pull/11384? They are generic and will be useful for any consumer.

Reply:
Done.  Thanks